tests/framework/installer/ceph_manifests_v1.0.go

```diff
@@ -1198,7 +1198,10 @@ spec:
     config:
       storeType: "` + settings.StoreType + `"
       databaseSizeMB: "1024"
-      journalSizeMB: "1024"`
+      journalSizeMB: "1024"
+  mgr:
```

Comment:
actually we don't want to update the v1.0.go test file. It would be ignored anyway in the 1.0 install. We need to update it in ceph_manifests.go

Reply:
done